src/site/content/en/learn/css/pseudo-elements/pseudo-elements.assess.yml

```diff
@@ -23,7 +23,7 @@ questions:
   - type: multiple-choice
     cardinality: "1"
     correctAnswers: "1"
-    stem: Psuedo-elements can be found in an HTML file.
+    stem: Pseudo-elements can be found in an HTML file.
```

Comment:
i struggle so hard to type this out correctly! not surprised it's misspelled here in yaml hehe. thanks!

Reply:
You are welcome and I think I know the struggle, as I seem to keep swapping the vowels even when deliberately trying not to :grin:.